src/js/features/search/makeGetSearchFields.thunk.ts

```diff
@@ -11,7 +12,10 @@ export const makeGetSearchFields = createAsyncThunk<
   { rejectValue: string; state: RootState }
 >('query/makeGetSearchFields', async (_, { rejectWithValue, getState }) => {
   return await axios
-    .get(searchFieldsUrl, { params: getState().metadata.selectedScope })
+    .get(searchFieldsUrl, {
+      headers: { ...makeAuthorizationHeader(getState().auth.accessToken) },
+      params: getState().metadata.selectedScope,
```

Comment:
Isn't this the same as using `scopedAuthorizedRequestConfig`?

Reply:
yes, forgot during a refactor.

Reply:
done